README.md

```diff
@@ -372,6 +372,29 @@ Then fetch the logs for that agent:
 kubectl logs brupop-agent-podname --namespace brupop-bottlerocket-aws 
 ```
 
+### Why are my bottlerocket nodes egressing to `https://updates.bottlerocket.aws`?
+
+The [Bottlerocket updater API](https://github.com/bottlerocket-os/bottlerocket/blob/develop/sources/updater/README.md)
+utilizes metadata from a public URL to query and check for updates.
+The URL is `https://updates.bottlerocket.aws` and the Bottlerocket's `apiclient` requires
+access to this endpoint in order to perform updates.
+
+Cluster nodes running in production environments with limited network access
+may not be able to reach this metadata endpoint
+and you may see failures when updates are available or an update status that appears "stuck".
+
+To troubleshoot and validate this, [access one of your Bottlerocket nodes on your cluster](https://github.com/bottlerocket-os/bottlerocket/blob/9adbe40c3fced559b2d05829e62902060c5ecc9c/README.md#exploration),
+and manually execute `apiclient update check`. If you see errors that indicate `Failed to check for updates`
+and network timeouts, ensure that your nodes on cluster have access to `https://updates.bottlerocket.aws`.
+
+If it is unacceptable to give your Bottlerocket nodes outside network access,
+you may consider creating a self managed proxy within the cluster
+that can periodically scrape the update metadata information from `https://updates.bottlerocket.aws`.
```

Comment:
metadata?
They would need to clone the metadata and the targets to make use of the update repo.

```suggestion
that can periodically scrape the update ? information from `https://updates.bottlerocket.aws`.
```